### ISSUE TITLE
autogenerate more apis

### ISSUE THREAD

**Issue by dynnamitt** [greghendershott/aws#67]
Wouldn't it be nice to have more services ?
I've been looking into elixirs implementation and they build the functions from aws' go-lang json.
see https://github.com/aws-beam/aws-codegen.

Yes I'll look into it one day myself once I've finished building all the mandatory games in the Realms book ;-)

**Comment by greghendershott** [comment]
I agree it would be interesting to try that approach, particularly because that's how -- if I recall correctly from looking at this some years ago -- that's how AWS themselves started to cope with their "API explosion" in lang wrappers like `boto`.

Although it sounds like a fun project for someone someday, I currently have no plans to do that, myself.

p.s. To the extent I use AWS these days, from within Racket apps, it is still just the services already supported in this library. (Even when I might use a few more services to create infrastructure in which the apps run, like EC2 or ECS or whatever, the apps themselves don't need a wrapper for those services. Also there are things like RDS where the app talks to e.g. Postgres in the usual ways, not through an AWS API.)
